### PR TITLE
feat: used linter for swift

### LINKED ISSUE #414: New Practice: Used Linter for Swift
## New Practice Proposal

Create a new practice that checks if is used linter for Swift.

### PR DESCRIPTION
**Pull request by Ogatron3000**
## Description

Add linter practice for Swift:
https://github.com/realm/SwiftLint 

## Motivation and Context

Resolves #414 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated the documentation accordingly.
- [ ] I have added new practice to practice list in README.md.
- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/practices/Swift/SwiftLintUsedPractice.ts
Comment (adelkahomolova) [inline src/practices/Swift/SwiftLintUsedPractice.ts:32]: add a new line at end of the file
```suggestion
}

```

File: src/practices/Swift/SwiftLintUsedPractice.spec.ts
Comment (adelkahomolova) [inline src/practices/Swift/SwiftLintUsedPractice.spec.ts:7]:
```suggestion
describe('SwiftLintUsedPractice', () => {
```

File: src/practices/Swift/SwiftLintUsedPractice.spec.ts
Comment (adelkahomolova) [inline src/practices/Swift/SwiftLintUsedPractice.spec.ts:51]:
```suggestion
});

```

**Review by adelkahomolova — CHANGES_REQUESTED** [review]

**Comment by adelkahomolova** [comment]
Thank you @Ogatron3000 for your PR! There are just a few small issues that need to be fixed. I appreciate your time and I hope you enjoyed contributing to our repository. :)

**Comment by Ogatron3000** [comment]
Fixed, thank you :)

**Comment by adelkahomolova** [comment]
Hi, @Ogatron3000 , thank you for the fixes. I'll just wait for approval from my colleague since we always want two approvals as we believe this is the best practice to keep a high quality of code.
If you'll want to continue in contributing in open-source, check our other issues on [dx-scanner](https://github.com/DXHeroes/dx-scanner/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc). If you're good at writing, (or you just want to self-education) check our open-source [Knowledge Base](https://developerexperience.io/) (here are the [issues](https://github.com/DXHeroes/knowledge-base-content/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) )
Take care of yourself in these complicated days! ❤️

**Review by adelkahomolova — APPROVED** [review]

**Review by Jakub-Vacek — APPROVED** [review]

**Comment by Ogatron3000** [comment]
Thank you @adelkahomolova, will do. Take care :)

**Pull request merged**
